modules/internal/ChapelIO.chpl

```diff
@@ -17,36 +17,310 @@
  * limitations under the License.
  */
 
-// ChapelIO.chpl
-//
+/*
+
+Basic types and utilities in support of I/O operation.
+ 
+Most of Chapel's I/O support is within the :mod:`IO` module.
+This automatically included module provides several basic
+types and routines that support the :mod:`IO` module.
+ 
+Writing and Reading
+~~~~~~~~~~~~~~~~~~~
+
+The :proc:`IO.writeln` function allows for a simple implementation
```

Comment:
You can force it to truncate with a "~" within the reference.  There's an example at the end of the Time module.